test/functional/Java22andUp/src/org/openj9/test/jep454/downcall/PrimitiveTypeTests1.java

```diff
@@ -301,13 +301,4 @@ public void test_printfFromDefaultLibWithMemAddr_LinkerOption_1() throws Throwab
 		MemorySegment formatSegmt = arena.allocateFrom("\n%d + %d = %d\n");
 		mh.invoke(formatSegmt, 15, 27, 42);
 	}
-
-	@Test
-	public void test_validateTrivialOption_1() throws Throwable {
```

Comment:
Why do we want to remove this test? Is it not still valid?

Reply:
There are two reasons for removing the test:
1) `Linker.Option.isTrivial()` (only true in downcall in JDK21) has been totally removed & replaced with `Linker.Option.critical()` in OpenJDK (since JDK22) which is only intended for the critical downcall when setting to true via `critical(true)`.
2) We have added more test cases intended for the heap arguments to verify the critical downcall via `critical(true)` rather then simply verifying the passed-in argument itself (no need to do that).

So the test case in JDK21 are outdated and meaningless in JDK22.